hello/README.md

```diff
@@ -1,7 +1,11 @@
-This is "Hello world" with assembly for Linux x86-64
+# Hello world
```

Comment:
Consider merging all examples into one directory, as discussed.

Reply:
in another pr